app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
```

Comment:
Place `self` on a new line, so opening `(` and closing `)` parentheses will not have arguments on their lines.